server/src/routes/comment.ts

```diff
@@ -0,0 +1,23 @@
+import { Request, Response, Router } from "express";
+
+import { createComment, getAllComments } from "../controller/comment";
+// import { validateUUID } from "../middleware/validateUUID";
+
+const router: Router = Router();
+
+router.post("/:itemId/:customerId", async (req: Request, res: Response) => {
```

Comment:
The post request it's not usually to a dynamic route, you should make the post request and send both id's via the req.body

Reply:
Also I don't see that you are calling any validation methods as a middleware